dts/arm/st/f0/stm32f0.dtsi

```diff
@@ -179,7 +179,8 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 			reg = <0x40005400 0x400>;
-			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x00200000>;
```

Comment:
Please provide more comments in the commit message why we are defining default clock sources (which is not a common practice). According to existing code, I can see:
```
	 * STM32F0/3 I2C's independent clock source supports only
	 * HSI and SYSCLK, not APB1. We force I2C clock source to SYSCLK.
	 * I2C2 on STM32F0 uses APB1 clock as I2C clock source
```